x/executionlayer/types/msgs.go

```diff
@@ -73,15 +73,8 @@ func (msg MsgExecute) GetSigners() []sdk.AccAddress {
 
 // MsgTransfer for sending deploy to execution engine
 type MsgTransfer struct {
-	TokenContractAddress string                    `json:"token_contract_address"`
-	FromPubkey           secp256k1.PubKeySecp256k1 `json:"from_pubkey"`
-	ToPubkey             secp256k1.PubKeySecp256k1 `json:"to_pubkey"`
-	TransferCode         []byte                    `json:"transfer_code"`
-	TransferArgs         []byte                    `json:"transfer_args"`
-	PaymentCode          []byte                    `json:"payment_code"`
-	PaymentArgs          []byte                    `json:"payment_args"`
-	GasPrice             uint64                    `json:"gas_price"`
-	Signer               sdk.AccAddress            `json:"signer"`
+	ToPubkey   secp256k1.PubKeySecp256k1 `json:"to_pubkey"`
```

Comment:
I recommend to using `crypto.Pubkey` interface type because for key type changeable. 
It has reason to use secp256k1 directly?

Reply:
@slave5vw It will be replaced to `sdk.AccAccount` when rebasing #59

Reply:
Will be change later.. related to nameservice.. If I change here.. will be so many conflict... 
I'll make a issue